### PR TITLE
improvements in velocity controller

### PR DESCRIPTION
**Pull request by CarlottaSartore**
This PR implements various improvements in the velocity control and the related update in the documentation.

### CODE REVIEW

File: library/simulink-library/MomentumVelocityControl/src/ComputeContactJacobianWithFeetContactFCN.m [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    J_c_dot_nu(1:6) = J_dot_nu_L;
+end
+
+% if right foot in contact
+if(feet_contact_status(2))
+    J_c(7:end,:)      = J_R;
+    J_c_dot_nu(7:end) = J_dot_nu_R;
+end
+
+end
```
Comment (gabrielenava) [inline library/simulink-library/MomentumVelocityControl/src/ComputeContactJacobianWithFeetContactFCN.m]: Missing newline

File: library/simulink-library/MomentumVelocityControl/src/ComputeReferences.m [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+L_int     = [L_int_lin; L_int_ang]; 
+L_star    = Ld-L_int; 
 
-    s_dot_star_k_1 = s_dot_des_k_1 - Gains.KP_Postural*(s_k- s_des_k); 
-end
+L_dot_star = L_dot_d -blkdiag(Gains.KD_CoM, Gains.KP_ang)*(L-Ld) -L_int;
+
+s_dot_star_k_1 = s_dot_des_k_1 - Gains.KP_Postural*(s_k-s_des_k); 
+
+end
```
Comment (gabrielenava) [inline library/simulink-library/MomentumVelocityControl/src/ComputeReferences.m]: also here missing newline

File: library/simulink-library/MomentumVelocityControl/src/ComputeReferencesController.m [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+%L_d(4:6)= L_ang_des;
+L_dot_d   = zeros(6,1); 
+L_d(1:3)  = M(1,1)*desired_pos_vel_COM(:,2)';
+int_L_ang    = zeros(3,1); 
+x_com_des    = desired_pos_vel_COM(:,1); 
+s_des_k      = joint_pos_desired; 
+s_dot_des_k  = zeros(size(s_des_k)); 
+
+
+end
```
Comment (gabrielenava) [inline library/simulink-library/MomentumVelocityControl/src/ComputeReferencesController.m]: same comment

File: library/simulink-library/MomentumVelocityControl/src/centroidalConversion.m [hunk 4]
```diff
@@ -0,0 +1,44 @@
+function centroidalDyn = centroidalConversion(DYNAMICS,FORKINEMATICS,STATE)
```
Comment (gabrielenava) [inline library/simulink-library/MomentumVelocityControl/src/centroidalConversion.m:1]: these centroidal -related functions could be moved in the `wbc` library folder. But for now it is ok to have them in the sources of the velocity controller, being the only user of the transformation.

**Review by gabrielenava — CHANGES_REQUESTED** [review]
I looked at the matlab functions and everything seems ok to me, but I cannot test the block at the moment. However overall seems ok, we can merge in `devel` after addressing my small ocmments.

File: library/simulink-library/MomentumVelocityControl/src/ComputeContactJacobianWithFeetContactFCN.m (hunk 1 above)
Comment (CarlottaSartore) [reply library/simulink-library/MomentumVelocityControl/src/ComputeContactJacobianWithFeetContactFCN.m]: done in 6c3d28fec228d23323ca3c7d07f2342666e69d7e

File: library/simulink-library/MomentumVelocityControl/src/ComputeReferences.m (hunk 2 above)
Comment (CarlottaSartore) [reply library/simulink-library/MomentumVelocityControl/src/ComputeReferences.m]: done in 6c3d28fec228d23323ca3c7d07f2342666e69d7e

File: library/simulink-library/MomentumVelocityControl/src/ComputeReferencesController.m (hunk 3 above)
Comment (CarlottaSartore) [reply library/simulink-library/MomentumVelocityControl/src/ComputeReferencesController.m]: done in 6c3d28fec228d23323ca3c7d07f2342666e69d7e

**Review by gabrielenava — APPROVED** [review]

**Pull request merged**